### DISCUSSION TITLE
Cloning the SSD

### DISCUSSION THREAD

**Discussion by Neophema** [gtxaspec/wz_mini_hacks#493] [Q&A]
I am trying to deploy some new (refurb) V3s.  The firmware was backreved so I brought them up to 139. They work fine with the app. 

I bought a couple of V3's back mid 2022; following the instructions I got them working fine, MRL and SSH. 

Using duplicate SSD's, the new cameras do not work with rtsp. I'm a little rusty on the configuration specifics; do they have or need something unique to each camera? I can't connect to them via MRL or ssh, only with the Wyze app.

**Comment by bjs-pdx** [comment]
I think if you actually cloned one from a working camera the first boot process creates some files on the SD card, specific to the camera.  You probably need a setup that hasn't been booted before on the card.
